### PR TITLE
Hide CI passed logs for yaml files checking

### PR DESCRIPTION
**Pull request by rfan1**
- Related ticket: https://progress.opensuse.org/issues/122923
- Verification run: n/a

### CODE REVIEW

**Comment by RoyCai7** [comment]
It looks like it works well. LGTM

File: tools/check_yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+success=1
+YAMLS=$(git ls-files schedule/ test_data/ | grep '.ya\?ml$' | xargs echo)
+
+if test -n "$YAMLS"; then \
```
Comment (grisu48) [inline tools/check_yaml:6]:
```suggestion
if test -n "$YAMLS"; then
```
If this is in an external bash file, then all the `\` at the end can be omitted.

Same applies to almost every line in this script.

File: tools/check_yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        export PERL5LIB=${PERL5LIB_} ; echo "$YAMLS" | xargs tools/test_yaml_valid | grep -Ev "has valid yaml syntax|has valid schema";\
+        else \
+        echo "No yamls modified.";\
+fi
+if test -n "$YAMLS"; then \
```
Comment (grisu48) [inline tools/check_yaml:11]: Isn't this check done twice and can be merged? :thinking: 

Line 6 and 11 look exactly the same to me and I don't see any changes to `$YAMLS` itself.

**Review by grisu48 — COMMENTED** [review]
Two suggestions. Let's remove the unnecessary `\` from the script!

File: tools/check_yaml (hunk 2 above)
Comment (rfan1) [reply tools/check_yaml:11]: You are right, let me revise my code

File: tools/check_yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+        which yamllint >/dev/null 2>&1 || (echo "Command 'yamllint' not found, can not execute YAML syntax checks" && exit 1) || success=0;\
+
+        echo "$YAMLS" | xargs yamllint -c .yamllint || exit 1 || success=0;\
+fi
+[ $success = 1 ] && echo "check_yaml SUCCESS" && exit 0
+exit 1
```
Comment (b10n1k) [inline tools/check_yaml:17]: That can be just `exit $success` right?

Comment (rfan1) [reply tools/check_yaml:14]: hmm, for shell, we may need to `exit 1` if any failure

Comment (grisu48) [reply tools/check_yaml:14]: Richard is right. `$success = 1` results in `exit 0`, so `exit $success` won't work.

**Review by grisu48 — APPROVED** [review]
LGTM!

**Pull request merged**